### DISCUSSION TITLE
選擇權的價性

### DISCUSSION THREAD

**Discussion by ChiuAlfredo** [ChiuAlfredo/options_day_data#27] [研究]
# 研究目標：價性對選擇權 Delta 值的影響

### 0. 研究資料
* 加權指數價格每一分鐘
* 選擇權價格每一分鐘
* real delta值:
$\delta = \frac{\delta 選擇權價格}{\delta 加權指數價格}$
* real gamma值:
$\gamma = \frac{\delta delta}{\delta 加權指數價格}$

### 1. ANOVA 概述
- **目的**：用於比較多組之間的均值差異
- **應用**：在本研究中，比較不同價性類別的 real Delta 值差異
- h_0:μ1 = μ2 = μ3 = ... = μk
- h_1:其中有一個不相等

### 2. 研究分組

以結算日當日為主

按照c跟p進行分組：
1. 買權
2. 賣權

按照選擇權的價性將數據分為以下幾組：
1. 深度價內 (Deep In-The-Money, DITM)
2. 價內 (In-The-Money, ITM)
4. 價平 (At-The-Money, ATM)
5. 價外 (Out-of-The-Money, OTM)
6. 深度價外 (Deep Out-of-The-Money, DOTM)

## 買權（call option）
$\frac{K}{S}$​ 範圍 | 價性分類
-- | --
$0.94 \leq \frac{K}{S} ​<0.97 $ | 深度價內 (DITM)
$0.97 \leq \frac{K}{S} ​<0.99 $ | 價內 (ITM)
$0.99\leq \frac{K}{S} <1.01 $ | 價平 (ATM)
$1.01\leq \frac{K}{S} ​<1.03 $ | 價外 (OTM)
$1.03 \leq \frac{K}{S} ​<1.06 $ | 深度價外 (DOTM)

## 賣權（put option）
$\frac{K}{S}$​ 範圍 | 價性分類
-- | --
$1.03 \leq \frac{K}{S} ​<1.06 $ | 深度價內 (DITM)
$1.01\leq \frac{K}{S} ​<1.03 $ | 價內 (ITM)
$0.99\leq \frac{K}{S} <1.01 $ | 價平 (ATM)
$0.97 \leq \frac{K}{S} ​<0.99 $ | 價外 (OTM)
$1.03 \leq \frac{K}{S} ​<1.06$ | 深度價外 (DOTM)

最後會變成有10組：

深度價內 (Deep In-The-Money, DITM)
DITM - 買權 (call)
DITM - 賣權 (put)
價內 (In-The-Money, ITM)
ITM - 買權 (call)
ITM - 賣權 (put)
價平 (At-The-Money, ATM)
ATM - 買權 (call)
ATM - 賣權 (put)
價外 (Out-of-The-Money, OTM)
OTM - 買權 (call)
OTM - 賣權 (put)
深度價外 (Deep Out-of-The-Money, DOTM)
DOTM - 買權 (call)
DOTM - 賣權 (put)

## 統計量
$F=\frac{MSB}{MSE}$

## 拒絕域
$F>F_\alpha(k-1,n-k).  \alpha=0.05$

### 4. 後續分析
- 如果 ANOVA 結果顯著，進行事後檢驗（Post-hoc test）
  - 目的：確定具體哪些價性類別之間存在顯著差異
  - 可能的方法：
    * Tukey's HSD (Honestly Significant Difference) test
    * Bonferroni correction
    * Scheffe's method

### 5. 預期結果
- 驗證價性對 Delta 值是否有顯著影響
- 確定不同價性類別之間 Delta 值的差異程度

**Comment by Will150143** [comment]
1.你的虛無假設是什麼？
2.C跟P分開嗎？還是價內的C跟P一起比
3.每一組會有幾檔？為什麼會定這個範圍？
4.用的是real delta還是bs delta？請標公式

**Reply by ChiuAlfredo** [reply]
- [x] 1.你的虛無假設是什麼？
- [x] 2.C跟P分開嗎？還是價內的C跟P一起比
- [] 3.每一組會有幾檔？為什麼會定這個範圍？
直接使用價性去做分別，可能會比幾檔更好

- [x] 4.用的是real delta還是bs delta？請標公式

**Comment by Will150143** [comment]
我想知道這個驗證的目的是什麼？
![image](https://github.com/ChiuAlfredo/options_day_data/assets/133984319/b07dbf3a-58b2-40e9-8262-1eaf9bd8f8ef)
這是元大期貨的介紹，你是想驗證現實情況不符合該論述嗎

**Reply by ChiuAlfredo** [reply]
對啊，如果理論是完全正確的話，價外p幾乎不會有在12.30後有上漲機會

**Comment by TsuYiii** [comment]
1.為何以0.94<k/s<0.97及1.03<k/s<1.06為深度價位與深度價內的區間呢？
2.如果ANOVA最終結果並不顯著，是否有其他研究方向呢？

**Reply by ChiuAlfredo** [reply]
1. 這是之前維中給的定義，因為如果更小的話，可能會沒有交易量
2. 不顯著的話就代表價性的差異是不會影響real delta
